polytracker/src/polytracker/main.cpp

```diff
@@ -96,6 +98,9 @@ void polytracker_start(func_mapping const *globals, uint64_t globals_count,
            "instrumentation.\n");
   }
 
+  get_polytracker_tdag().open_file(fileno(stdout), "/dev/stdout");
+  get_polytracker_tdag().open_file(fileno(stderr), "/dev/stderr");
+
```

Comment:
I suggest we guard these statements and check that the return of fileno is >= 0 (in case they are closed for some reason).